tests/sai_qualify/sai_infra.py

```diff
@@ -76,10 +79,42 @@ def sai_test_env_check(creds, duthost, ptfhost, request):
         logger.info("Test env check is failed in previous check. \
             Fails this check directly.")
         raise Exception("SAI Test env error.")
-
     check_test_env_with_retry(creds, duthost, ptfhost, request)
 
 
+@pytest.fixture(scope="module")
+def start_warm_reboot_watcher(creds, duthost, request, ptfhost, create_sai_test_interface_param):
+    # install apscheduler before running
+    logger.info("start warm reboot watcher")
+    logging.getLogger("apscheduler.executors.default").propagate = False
+    logging.getLogger("apscheduler.executors.default").setLevel(logging.ERROR)
+    ptfhost.shell("touch {}".format("/tmp/warm_reboot"))
+    ptfhost.shell("echo  > {}".format("/tmp/warm_reboot"))
```

Comment:
for the running info, please add more logs for the operations, like created file for any purpose.